src/algorithms.jl

```diff
@@ -5216,6 +5216,11 @@ struct SofSpa10 <: OrdinaryDiffEqPartitionedAlgorithm end
 # Nyström methods
 
 """
+Improved Runge-Kutta-Nyström method of order three, which minimizes the amount of evaluated functions in each step. Fixed time steps only..
```

Comment:
```suggestion
"""
    IRKN3

Improved Runge-Kutta-Nyström method of order three, which minimizes the amount of evaluated functions in each step. Fixed time steps only.
```
Could you please add the name of the struct you describe in each docstring as in the suggestion above? Indenting it by 4 spaces lets it be rendered as code.